src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -77,12 +99,19 @@
     module.exports.{@service.serviceName} = {@service.localName};
   @end
 
+  module.exports.types = {@version}Protos;
+
+  module.exports.{@version} = {};
```

Comment:
Ah actually it's only necessary for multiple services since `vX` is undefined but we need to assign to `vX.serviceShortname`.

Fixed!